website/pages/commands/tls/cert.mdx

```diff
@@ -0,0 +1,83 @@
+---
+layout: commands
+page_title: 'Commands: TLS Cert Create'
+sidebar_title: cert
+---
+
+# Consul TLS Cert Create
+
+Command: `consul tls cert create`
+
+The `tls cert create` command is used to create certificates for your Consul TLS
```

Comment:
Do we potentially want to link out that this is for [Securing Agent Communication](https://learn.hashicorp.com/tutorials/consul/tls-encryption-secure) so that there is clarity on the usecase?

Reply:
This PR is just for restructuring, so maybe that's a change that can be discussed and made in the future

Reply:
Okay, will hold off :)